src/HPRT/toolkit/models.py

```diff
@@ -308,12 +288,43 @@ def get_labels(self):
         labels = {}
         json_data = open(os.path.join('static', "questions.json"), 'r')
         hp2_list = json.load(json_data)[8]['questions']
-        i = 11
-        for index in hp2_list:
+        for i, index in enumerate(hp2_list, start=1):
             labels[index['id']] = str(i) + '. ' + index['body']
-            i+=1
         json_data.close()
         return labels
 
 
-        
+class GeneralHealth(models.Model):
+    """ Abstract model of General Health/Physical Functioning questionnaire. """
+    patient = models.ForeignKey(Patient, on_delete = models.CASCADE)
+    doctor = models.ForeignKey(Doctor, on_delete = models.CASCADE)
+    date = models.DateField()
+    gh1 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = Poor"),(2,"2 = Fair"),(3,"3 = Good"),(4,"4 = Very Good"),(5,"5 = Excellent")])
```

Comment:
Same here - could someone leave 'choose one' as a valid choice?

Reply:
Nope. Form requires choice of a valid option.